### ISSUE TITLE
Improve site-wide surveys - child, demographic

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#489]
**TL;DR**: Review and revamp the child and demographic surveys after considering what information is most important to collect as well as family experience.

**Child characteristic options**: Right now we have a small set of checkboxes for various characteristics that parents can select from when registering a child - ASD, multiple birth, deaf, hard of hearing, dyslexic. It would be useful to have a more comprehensive set of options, but when we started listing conditions/characteristics off the tops of our heads we realized we don't have a good idea of the relative frequencies of various conditions -- or their relative frequencies in specialized research - to put together an appropriate set of options (that won't leave parents wondering why e.g. Down Syndrome is there but another trisomy isn't). This would be a great project for someone new to Lookit to take on - figure out a principled way to build a list, survey experimenters and/or families, etc., and propose an extension.

**Demographic survey questions/options**: The family demographic survey is also somewhat haphazard - it definitely provides useful information but no individual question has been considered as carefully as it might ideally be (and there may be questions left off). E.g., should we really be querying annual income in USD in increments of 10K up to 200K? What bins would be most appropriate? Do families generally interpret the question in a consistent same way or do we need to clarify the wording? Could we switch to a gender question that isn't literally "other"-ing both on the demographic survey and for children? What is best practice there - offering a custom write-in, just offering non-binary? Another great project for someone to take on and come back with recommendations.

**Comment by kimberscott** [comment]
Researchers at Western Sydney noted the demographic survey race options aren't applicable in Australia (and likely in most other countries), among other US-centric questions there. Possibly want to customize what we ask based on country, OR look into if anyone has good questions that get at race / ethnicity crossculturally (lots of people must deal with this, right?)
